how-to/how-to-use-rootless-kata-with-podman.md

```diff
@@ -0,0 +1,108 @@
+```bash
+sudo usermod -a -G kvm $USER
+```
+
+#### 3. Reboot
+Reboot the system for the first two steps to take effect. (*Reboot is required for SELinux change, logging out and logging back is enough for the user change to work*)
+```bash
+# To check changes took effect
+$ getenforce
+Disabled
+
+$ groups
+username kvm
+```
+
+#### 4. Disable vhost-net
+In the Kata configuration file, disable vhost-net, by uncommenting out the `disable_vhost_net = true` line <br>
+```bash
+# if the configuration.toml file already exists in /etc and is owned by user, skip these two lines
+sudo mkdir -p /etc/kata-containers/
+sudo install -o ${USER} -g root -m 0640 /usr/share/defaults/kata-containers/configuration.toml /etc/kata-containers
+
+sudo sed -i -e 's/^#disable_vhost_net = true/disable_vhost_net = true/' /etc/kata-containers/configuration.toml
+```
+
+#### 5. Modify Kata image permissions
+The Kata images needs to be accessible by the user, so change the permissions of the image directory to be owned by the user, instead of root.
+```bash
+sudo chown -R ${USER}:root /usr/share/kata-containers
+```
+#### 6. Set up Podman rootless configuration
+If `libpod.conf` does not exist in `~/.config/containers/`:
+```bash
+mkdir -p ~/.config/containers
+sudo install -o ${USER} -g ${USER} -m 0640 /usr/share/containers/libpod.conf ~/.config/containers/
+```
+By default the `tmp_dir` in `libpod.conf` is set to `/var/run/libpod`, however that is not accessible by a user, so change to the rootless runtime directory: `"/run/user/1000/libpod/tmp"`
+```bash
+sed -i -e "s|^tmp_dir = .*$|tmp_dir = \"$XDG_RUNTIME_DIR/libpod/tmp\"|" ~/.config/containers/libpod.conf
+```
+
+#### 7. Add Kata-runtime to Podman configuration file (optional)
+Add the Kata-runtime to the Podman configuration file to avoid passing the fully qualified Kata binary path <br> `--runtime=/usr/local/bin/kata-runtime` for every instance ran. <br>
+
+```bash
+# the runtime table is the final option in the config file, so append the Kata-runtime with the path(s) to the Kata-runtime binary
+echo "kata = [\"/usr/local/bin/kata-runtime\"]" >> ~/.config/containers/libpod.conf
+```
+*NOTE: The runtime name (`kata` in this case) will still need to be passed into the command with every Podman call: `--runtime=kata` **unless** `kata` is specified as the default in `libpod.conf`*
+
+```bash
+# to make Kata the default runtime with Podman (EXTRA OPTIONAL)
+sed -i -e 's/^runtime = \"runc\"/runtime = \"kata\"/' ~/.config/containers/libpod.conf
+```
+
+## Running Kata + Podman rootless
+```bash
+# assuming Kata is not set to be the default runtime
+podman run --rm --runtime=kata alpine date
+```
+
+*Note: enable [debug](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#enable-full-debug) in Kata to get runtime journald logs, and pass `--log-level=debug` to see Podman logs
+
+## Possible Errors (*mostly if building from source)*
```

Comment:
It might be worth changing this to:

```
## Appendix: Possible Errors

If you are building from source you may encounter the following errors.
```